### PR TITLE
feat: field groups props in editor layout [DANTE-173]

### PR DESCRIPTION
**Pull request by Silhoue**
## Summary

Add types for `groupControls` and `editorLayout` properties of the `EditorInterface`.

## Checklist (check all before merging)

- [x] Both unit and integration tests are passing
- [x] There are no breaking changes
- [x] Changes are reflected in the documentation

### CODE REVIEW

File: lib/entities/editor-interface.ts [hunk 1]
```diff
@@ -72,6 +93,10 @@ export type EditorInterfaceProps = {
    * Array of fields and it's associated widgetId
    */
   controls?: Control[]
+  /**
+   * Array of groups and it's associated widgetId
```
Comment (IBratWurst) [inline lib/entities/editor-interface.ts:97]: Probably cloned from above
```suggestion
   * Array of groups and their associated widgetId
```

File: lib/export-types.ts [hunk 2]
```diff
@@ -41,7 +41,9 @@ export type {
   EditorInterface,
   EditorInterfaceProps,
   Control,
+  GroupControl,
   Editor,
+  EditorLayoutItem,
```
Comment (IBratWurst) [inline lib/export-types.ts:46]: Should `FieldItem` also be exported?

File: lib/entities/editor-interface.ts (hunk 1 above)
Comment (Silhoue) [reply lib/entities/editor-interface.ts]: fixed

File: lib/export-types.ts (hunk 2 above)
Comment (Silhoue) [reply lib/export-types.ts:46]: added

**Review by veu — APPROVED** [review]

**Review by IBratWurst — APPROVED** [review]

**Pull request merged**